src/fairchem/core/models/gemnet_oc/gemnet_oc.py

```diff
@@ -234,6 +234,7 @@ def __init__(
         num_elements: int = 83,
         otf_graph: bool = False,
         scale_file: str | None = None,
+        raise_on_scale_load_error: bool = True,
```

Comment:
Is option necessary? Seems like it should just fail if you have partially loaded scaling factors or if the module naming is off

Reply:
yeah, i left it default True. but to be backwards compatible it can be set to False. im down to make it just hard fail!

Reply:
if you miss one layer , maybe its not that big of a deal?

Reply:
i've talked to myself some more, lets take it out! 🎉